src/test/java/com/emc/mongoose/storage/driver/pravega/integration/PravegaEventReadTest.java

```diff
@@ -41,51 +42,56 @@ public static void tearDownClass()
 	public void testEventRead()
 			throws Exception {
 		/* writing */
-		final String scopeName = "Scope";
-		final String streamName = "Stream";
-		final URI controllerURI = URI.create("tcp://127.0.0.1:9090");
-		final String routingKey = "RoutingKey";
-		final String testEvent = "TestEvent";
-		final int readerTimeoutMs = 100;
-		final StreamManager streamManager = StreamManager.create(controllerURI);
-		final boolean scopeIsNew = streamManager.createScope(scopeName);
-		StreamConfiguration streamConfig = StreamConfiguration.builder()
-				.scalingPolicy(ScalingPolicy.fixed(1))
-				.build();
-		final boolean streamIsNew = streamManager.createStream(scopeName, streamName, streamConfig);
+		val scopeName = "Scope";
+		val streamName = "Stream";
+		val controllerURI = URI.create("tcp://127.0.0.1:9090");
+		val routingKey = "RoutingKey";
+		val testEvent = "TestEvent";
+		val readerTimeoutMs = 100;
+		val streamManager = StreamManager.create(controllerURI);
+		val scopeIsNew = streamManager.createScope(scopeName);
+		val streamConfig = StreamConfiguration.builder()
+			.scalingPolicy(ScalingPolicy.fixed(1))
```

Comment:
We have deleted a Pravega Constant SEGMENT_AMOUNT and here we just use 1. this one defines the amount of segments. Why wouldnt we use a constant for this? or the name scaling policy is obvious enough?